miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/errors/MiniAppErrors.kt

```diff
@@ -0,0 +1,9 @@
+package com.rakuten.tech.mobile.miniapp.errors
+
+/** Type of errors for access token fetch **/
+@Suppress("MatchingDeclarationName")
+enum class AccessTokenErrorType(val errorKey: String) {
+    AudienceNotSupportedError("AudienceNotSupportedError"),
+    ScopesNotSupportedError("ScopesNotSupportedError"),
+    AuthorizationFailureError("AuthorizationFailureError");
```

Comment:
The spec of `Authorization failure` declared that it is a custom error message sent from hostapp.
If we go only with `AccessTokenErrorType.errorKey` then the hostapp user cannot pass the error message so

we can create a `MiniAppAccessTokenError` class which contains `error: AccessTokenErrorType = OtherError` and `message: String = ""`.

When user does not pass the `message` which is empty as default, we will let the errorCallback send `AccessTokenErrorType.errorKey`.